abb

```diff
@@ -21,7 +21,7 @@
 # defaults:
 
 # using proto (http|https|ssh)
-proto=https
+proto=ssh
```

Comment:
No reason to change it, because you always can point 
"proto" in your .abbrc file

I tried to replace it on clean system with empty abbrc file
and got this:

Warning: Permanently added 'github.com,192.30.252.130' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.